.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: csharp
+solution: kin-base-csharp.sln
+
+mono: none
+
+matrix:
+  include:
+    - os: linux
+      dist: xenial
+    - os: osx
+    # - os: windows , very expiremental, does not work yet
```

Comment:
Out of curiosity, what's not working in windows? Funny..

Reply:
Windows is still in pre-alpha for travis, when turned it on it tried to run some unix commands (sudo etc), didnt really investigate much